python/pyspark/ml/tests.py

```diff
@@ -1018,6 +1018,48 @@ def test_parallel_evaluation(self):
         cvParallelModel = cv.fit(dataset)
         self.assertEqual(cvSerialModel.avgMetrics, cvParallelModel.avgMetrics)
 
+    def test_expose_sub_models(self):
```

Comment:
Nice tests.  Can you make one addition: Test the copy() method to make sure it copies the submodels.